templates/app/versions/v1.0.0/app.yml

```diff
@@ -0,0 +1,175 @@
+              AWS: !GetAtt AdministrationRole.Arn
+            Action:
+              - sts:AssumeRole
+      Path: /
+      Policies:
+      - PolicyName: ExecutionRolePolicy
+        PolicyDocument:
+          Version: "2012-10-17"
+          Statement:
+              - Sid: StackSetRequiredPermissions
+                Effect: Allow
+                Action:
+                  - cloudformation:*
+                  - s3:*
+                  - sns:*
+                Resource: "*"
+              - Sid: ManageKMSKeys
+                Effect: Allow
+                Action:
+                  - kms:*
+                Resource: "*"
+              - Sid: ManageECRRepos
+                Effect: Allow
+                Action:
+                  - ecr:DescribeImageScanFindings
+                  - ecr:GetLifecyclePolicyPreview
+                  - ecr:CreateRepository
+                  - ecr:GetDownloadUrlForLayer
+                  - ecr:GetAuthorizationToken
+                  - ecr:ListTagsForResource
+                  - ecr:ListImages
+                  - ecr:DeleteLifecyclePolicy
+                  - ecr:DeleteRepository
+                  - ecr:SetRepositoryPolicy
+                  - ecr:BatchGetImage
+                  - ecr:DescribeImages
+                  - ecr:DescribeRepositories
+                  - ecr:BatchCheckLayerAvailability
+                  - ecr:GetRepositoryPolicy
+                  - ecr:GetLifecyclePolicy
+                  - ecr:TagResource
+                Resource: "*"
+
+  DNSDelegationRole:
+    Type: AWS::IAM::Role
+    Condition: DelegateDNS
+    Properties:
+      RoleName: !Ref DNSDelegationRoleName
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS:  !Sub arn:aws:iam::${AWS::AccountId}:root
+            Action:
+              - sts:AssumeRole
+          - Effect: Allow
+            Principal:
+              AWS: !Split
+                - ','
+                - !Sub
+                    - 'arn:aws:iam::${inner}:root'
+                    - inner: !Join
+                      - ':root,arn:aws:iam::'
+                      - Ref: "AppDNSDelegatedAccounts"
+            Action:
+              - sts:AssumeRole
+      Path: /
+      Policies:
+      - PolicyName: DNSDelegationPolicy
+        PolicyDocument:
+          Version: "2012-10-17"
+          Statement:
+              - Sid: HostedZoneReadRecords
+                Effect: Allow
+                Action:
+                  - route53:Get*
+                  - route53:List*
+                Resource: "*"
+              - Sid: HostedZoneUpdate
+                Effect: Allow
+                Action:
+                  - route53:ChangeResourceRecordSets
+                Resource:
+                  - !Sub arn:${AWS::Partition}:route53:::hostedzone/${AppHostedZone}
+                  - !Sub arn:${AWS::Partition}:route53:::hostedzone/${AppDomainHostedZoneID}
```

Comment:
you only changed this line right? just to double check

Reply:
Yes. That's right.